proto/steward.proto

```diff
@@ -19,6 +19,7 @@ import "cellar_v2.proto";
 service ContractCall {
     // Handles simple contract call submission
     rpc Submit(SubmitRequest) returns (SubmitResponse) {}
+    rpc Status(StatusRequest) returns (StatusResponse) {}
```

Comment:
In retrospect it's kinda weird that the `Status` call is in the `ContractCall` service but I don't know if it's worth changing.